test/parallel/test-crypto-binary-default.js

```diff
@@ -626,7 +626,10 @@ common.expectsError(
 
   rsaSign.update(rsaPubPem);
   const rsaSignature = rsaSign.sign(rsaKeyPem, 'hex');
-  const expectedSignature = fixtures.readKey('rsa_public_sha1_signature_signedby_rsa_private.sha1', 'hex')
```

Comment:
It looks like this line wouldn't lint clean, where was the overlong line introduced? If possible, can you fixup the original commit that introduced the line? Breaking `git bisect` with lint isn't the end of the world, but if its possible to avoid that would be good.

Reply:
I went back and altered the commits that introduced lint failures.